tests/tasks/test_bgpstatemonitortask.py

```diff
@@ -192,6 +192,30 @@ async def test_oper_down_creates_event(self, task):
         assert event.remote_as == 20
         assert event.peer_uptime == 1000000
 
+    @pytest.mark.asyncio
+    @pytest.mark.parametrize(
+        "task",
+        ["general-bgp-oper-down-short", "cisco-bgp-oper-down-short", "juniper-bgp-oper-down-short"],
+        indirect=True,
+    )
+    async def test_oper_down_does_not_create_event_if_uptime_less_than_specified(self, task):
+        """Tests that an event should not be made if a BGP connection to a device reports that their oper_state has
+        changed from established to something else, but the uptime is less than a specified time
+        """
+        peer_address = IPv4Address("10.0.0.1")
+        # set initial state
+        if task.device_state.bgp_style == "juniper":
+            task.device_state.bgp_peer_admin_status = {peer_address: "running"}
+        else:
+            task.device_state.bgp_peer_admin_status = {peer_address: "start"}
+        task.device_state.bgp_peer_oper_states = {peer_address: "established"}
+        await task.run()
+        # check if state has been updated to reflect state defined in .snmprec
+        assert task.device_state.bgp_peer_oper_states[peer_address] != "established"
+        # check that the correct event has been created
```

Comment:
Surely you mean:
> Verify that no event was created

?

Reply:
I think you looked at an old commit, this was fixed in https://github.com/Uninett/zino/pull/133/commits/c8f2f4ffba2f921ee14c2dc021486677addca8d3